Snowdog_Components/components/Organisms/dashboard/nav/_nav.scss

```diff
@@ -169,6 +169,11 @@
                 display: none;
             }
         }
+
+        & > strong {
+            display: block;
+            max-width: 20ch;
```

Comment:
max-width: 20ch creates always 2 line of text for longer items. Better att padding right: $spacer--medium, but please create variable for it

Reply:
Actually I had to add bigger right padding, because there was same issue as reported.